### DISCUSSION TITLE
47회차 반성문

### DISCUSSION THREAD

**Discussion by jojaeng2** [BDD-CLUB/challenge-every-week-posting#492] [5. 회고]
안녕하세요. 처음으로 반성문을 쓰네요. 
이번주는 공부를 하나도 하지 못했습니다. 반성합니다.
약간의 TMI 이지만 저는 뭔가에 꽂히면 그것만 하는데 요즘은 롤에 빠져서, 이번주는 매일 4시간씩 자고 일과 게임만 했습니다. 
일주일 동안 롤을 거의 90판을 했네요
다음주에 회복할 수 있을지는 모르겠습니다. 
아마 게임이 질릴때 쯤 다시 공부를 잡을 것 같은데, 빨리 회복해보겠습니다

**Comment by 02ggang9** [comment]
ㄷㄷ.. 한번씩 풀어주는 것도 좋은 것 같습니다~!
